src/utils/validation.ts

```diff
@@ -14,3 +14,7 @@ export const zPassword = z.string().refine(
     message: 'Le mot de passe doit contenir au moins 8 caractères, une lettre minuscule, une lettre majuscule et un chiffre.',
   }
 );
+
+export const emailSchema = z.string().trim().toLowerCase().email('Invalid email address');
+
+export const sanitizeEmail = (email: string): string => emailSchema.safeParse(email)?.data ?? email;
```

Comment:
Attention, il me semble que safeParse n'empêche pas toujours le schéma de lancer des exceptions (je crois que c'est lié aux transforms)

Reply:
Pas noté de pb malgré les emails problématiques donc je laisse